Tests/SwiftSyntaxBuilderTest/ProtocolDeclTests.swift

```diff
@@ -11,9 +11,10 @@ final class ProtocolDeclTests: XCTestCase {
     })
     let functionSignature = FunctionSignature(input: input, output: returnType)
 
-    // FIXME: We need to add the `modifiersBuilder` with a non-empty value, otherwise will the builder omit newline.
-    let functionDecl = FunctionDecl(identifier: .identifier("buildDeclList"), signature: functionSignature, modifiersBuilder: { TokenSyntax.public })
-    let buildable = ProtocolDecl(modifiers: TokenSyntax.public, identifier: .identifier("DeclListBuildable"), members: functionDecl)
+    let buildable = ProtocolDecl(identifier: "DeclListBuildable", attributesBuilder: { TokenSyntax.public }, membersBuilder: {
```

Comment:
We cannot remove the parameter `membersBuilder`. Otherwise we get the warning `Backward matching of the unlabeled trailing closure is deprecated; label the argument with 'membersBuilder' to suppress this warning`

Reply:
I think that’s reasonable 👍